pkg/apiserver/api_installer.go

```diff
@@ -204,10 +204,11 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	}
 	versionedStatus := indirectArbitraryPointer(versionedStatusPtr)
 	var (
-		getOptions     runtime.Object
-		getOptionsKind string
-		getSubpath     bool
-		getSubpathKey  string
+		getOptions          runtime.Object
+		versionedGetOptions runtime.Object
+		getOptionsKind      string
+		getSubpath          bool
+		getSubpathKey       string
 	)
 	if isGetterWithOptions {
 		getOptions, getSubpath, getSubpathKey = getterWithOptions.NewGetOptions()
```

Comment:
Yes, it's currently only implemented by the pod's LogREST storage. It's purpose is to allow the implementing REST storage to receive additional parameters in its get request. The getOptions object is meant to be introspected to correlate request parameters to options object fields, the subpath and subpathkey make it possible to set one of the fields to correspond to the path parameter in the request url.